client/reader/feed-stream/index.jsx

```diff
@@ -140,6 +151,7 @@ var FeedStream = React.createClass( {
 
 		return (
 			<FollowingStream { ...this.props } listName={ this.state.title } emptyContent={ emptyContent } >
+				{ this.props.showBack ? <HeaderCake isCompact={ false } onClick={ this.goBack } /> : null }
```

Comment:
+1 for using a ternary. Saw another PR earlier today that had its check value being rendered to the page.